api/views.py

```diff
@@ -1,21 +1,36 @@
+import jwt
 from django.utils import timezone
 from drf_yasg.openapi import Parameter, IN_QUERY, TYPE_STRING
 from drf_yasg.utils import swagger_auto_schema
 from rest_framework import status
 from rest_framework.exceptions import ValidationError
-from rest_framework.generics import ListAPIView, RetrieveAPIView, get_object_or_404
+from rest_framework.generics import ListAPIView, RetrieveAPIView, get_object_or_404, DestroyAPIView
 from rest_framework.response import Response
 from rest_framework_simplejwt.views import TokenObtainSlidingView, TokenRefreshSlidingView, TokenVerifyView
 
 from api.serializers import CheckBalanceSerializer, SociProductSerializer, ChargeSociBankAccountDeserializer, \
     PurchaseSerializer
 from api.view_mixins import CustomCreateAPIView
-from economy.models import SociBankAccount, SociProduct, Purchase
+from economy.models import SociBankAccount, SociProduct, Purchase, SociSession
 
 
 class CustomTokenObtainSlidingView(TokenObtainSlidingView):
     swagger_schema = None
 
+    def post(self, request, *args, **kwargs):
+        response = super().post(request, *args, **kwargs)
+        self._start_new_soci_session(token=response.data['token'])
```

Comment:
Should we have a separate view for this? My initial idea was that a session could be started automatically when the user authenticates, but now I'm starting to think that we should keep these two features separate.